rts/Lua/LuaOpenGL.cpp

```diff
@@ -5661,5 +5666,69 @@ int LuaOpenGL::GetMapRendering(lua_State* L)
 	return 0;
 }
 
+
+/**
+ * @function gl.ObjectLabel labels an object for use with debugging tools
+ * @param objectTypeIdentifier GLenum Specifies the type of object being labeled.
+ * @param objectID GLuint Specifies the name or ID of the object to label.
+ * @param label string A null-terminated string containing the label to be assigned to the object.
+ * @treturn nil
+ */
+int LuaOpenGL::ObjectLabel(lua_State* L) {
+	GLenum identifier = (GLenum)luaL_checkinteger(L, 1);
+	if (identifier != GL_BUFFER && identifier != GL_SHADER && identifier != GL_PROGRAM &&
+		identifier != GL_VERTEX_ARRAY && identifier != GL_QUERY && identifier != GL_PROGRAM_PIPELINE &&
+		identifier != GL_TRANSFORM_FEEDBACK && identifier != GL_TEXTURE && identifier != GL_RENDERBUFFER &&
+		identifier != GL_FRAMEBUFFER) {
+		return 0;
+	}
```

Comment:
I found this snippet in the gameside PR:
```lua
local GL_PROGRAM = 0x82E2
gl.ObjectLabel(GL_PROGRAM, gl_program_id, self.shaderName)
```
Sounds bad if the game needs to know those magic `0x82E2` constants to use the function. Would be good if there were either appropriate named constants in `LuaConstGL.cpp` (`GL.PROGRAM` etc) or if this accepted a string ("program" etc).